File: step24_lambda_layers/Example01_lambda_with_multiple_layers/lambda-fns/lambda.ts
const random_name = require('/opt/randomName');
const axios = require('axios');

exports.handler = async (event: any, context: any) => {

    const name = random_name.getName();
    const result = await axios.get('https://jsonplaceholder.typicode.com/todos/1');

    console.log("Random Name ==>", name);
    console.log("Random Fetch ==>", result.data);

    return {
        statusCode: 200,
        headers: { "Content-Type": "application/json" },
        body: {
            randomName: name,
            randomFetch: result.data,
        },
    }

}

File: step24_lambda_layers/Python/Example01_lambda_with_multiple_layers/lambda-fns/lambda.py
import requests
import json
import randomName

def handler(event, context):    

    name = randomName.getName()    
    result = requests.get('https://jsonplaceholder.typicode.com/todos/1')    
    
    print("Random Name ==>", name)
    print("Random Fetch ==>", result.json())

    return {
        'statusCode': 200,
        'headers': { "Content-Type": "application/json" },
        'body': {
            'randomName': name,
            'randomFetch': result.json(),
        },
    }

 

